NightscoutUploadKit/DeviceStatus/PredictedBG.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PredictedBG.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 8/8/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct PredictedBG {
+    let values: [Double]
+    let cob: [Double]?
+    let iob: [Double]?
+
+    public init(values: [Double], cob: [Double]? = nil, iob: [Double]? = nil) {
```

Comment:
For the start date of the points, the timestamp on the parent structure (LoopSuggested, or LoopEnacted) is used.  I'm thinking that for LoopSuggested, it's going to usually match? i.e. the start time of the forecasted points is going to match recommendedDate on TempBasalRecommendation?  If not, I can add a timestamp that is the actual start time of the forecasted points to this structure, and submit a patch to NS to use it.